contrib/release/post-release.sh

```diff
@@ -65,10 +65,11 @@ main() {
 
     git checkout -b pr/$version-digests $version
     ${DIR}/pull-docker-manifests.sh "$@"
+    logrun make -C Documentation update-helm-values
     logecho
     logecho "Check that the following changes look correct:"
     # TODO: Make this less interactive when we have used it enough
-    git add --patch install/kubernetes
+    git add --patch install/kubernetes Documentation/
```

Comment:
Any helm fixes should be prompted on PRs updating helm files, no?

Reply:
yes, this piece just automates the process when preparing releases. I hit this because usually I prepare releases just by running these scripts and they post the PRs automatically and we just have to click approve + merge. However this time around, I got some CI faliures because I hadn't run this command. Goal here is just to push the extra step into the scripting for release automation so we don't have to worry about it.

Reply:
>However this time around, I got some CI faliures because I hadn't run this command. 

Does that mean some helm issues weren't flagged on the original PRs that introduced those changes, and slipped through the cracks?

Reply:
No, this is more to do with how this script modifies the Helm values (eg docker container digests) a few lines above, but now we also publish those values in the docs, which we didn't use to do.